yarn.lock

```diff
@@ -2505,6 +2505,11 @@
   resolved "https://registry.yarnpkg.com/@spectrum-css/vars/-/vars-2.2.0.tgz#f266d62146cf60d560d9fb68f1cf3c96441229c1"
   integrity sha512-cYGvOlgfKpx1tGT822l7/SZTzonbIJIj/mJUWJkzMjBYt3z/swx2t1vupWoPZMTt1FBlgMhR1C6Fup7w/68qzA==
 
+"@spectrum-css/vars@^2.3.0":
```

Comment:
Odd there was no entry for `@spectrum-css/vars` before this?

Reply:
Good catch here! There was another entry that was still pointing to the previous version elsewhere in the lock but was able to clean it up with this note.